_1327/settings.py

```diff
@@ -73,12 +74,19 @@
 AUTHENTICATION_BACKENDS = [
 	'django.contrib.auth.backends.ModelBackend',
 	'guardian.backends.ObjectPermissionBackend',
+	'_1327.user_management.authentication.IPRangeAuthorizationBackend',
 ]
 
 # needed by django-guardian library
 ANONYMOUS_USER_ID = -1
 GUARDIAN_RAISE_403 = True
 
+# Anonymous users in one of the given IP ranges are automatically assumed to be in the associated group.
+# Only the first match will be used.
+ANONYMOUS_IP_RANGE_GROUPS = {
+	# Example: '127.0.0.0/8': 2,
```

Comment:
could you do this by group name? we have defined `UNIVERSITY_GROUP_NAME` in the settings. and if we need a custom group to be added it's still easier to recognize by name.